gdk4/src/dmabuf_texture_builder.rs

```diff
@@ -0,0 +1,66 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+
+use crate::{DmabufTextureBuilder, Texture};
+
+impl DmabufTextureBuilder {
+    #[doc(alias = "gdk_dmabuf_texture_builder_build")]
+    #[must_use = "The builder must be built to be used"]
+    #[allow(clippy::missing_safety_doc)]
+    pub unsafe fn build(self) -> Result<Texture, glib::Error> {
+        let mut error = std::ptr::null_mut();
+
+        let result = ffi::gdk_dmabuf_texture_builder_build(
+            self.to_glib_none().0,
+            None,
+            std::ptr::null_mut(),
+            &mut error,
+        );
+        if error.is_null() {
+            if result.is_null() {
+                Err(glib::Error::new(
+                    crate::DmabufError::UnsupportedFormat,
+                    "Unsupported format",
+                ))
+            } else {
+                Ok(from_glib_full(result))
+            }
+        } else {
+            Err(from_glib_full(error))
+        }
+    }
+
+    #[doc(alias = "gdk_dmabuf_texture_builder_build")]
+    #[must_use = "The builder must be built to be used"]
+    #[allow(clippy::missing_safety_doc)]
+    pub unsafe fn build_with_release_func<F: FnOnce() + 'static>(
```

Comment:
```suggestion
    pub unsafe fn build_with_release_func<F: FnOnce() + Send + 'static>(
```

I think. Texture are supposed to be thread-safe or not? Same change I guess for the GL texture?

Reply:
They are thread safe yes, would do the same for GL textures